tests/acceptance/02_classes/02_functions/classesmatching_inherit.cf

```diff
@@ -0,0 +1,38 @@
+bundle agent __main__
```

Comment:
Unindent everything after this line, once

Reply:
haha! says the guy who worked on a cfengine policy language parser/formatter. ;)